libethcore/Transaction.cpp

```diff
@@ -153,8 +164,11 @@ void TransactionBase::streamRLP(RLPStream& _s, IncludeSignature _sig, bool _forE
 
 	if (_sig)
 	{
+		if (!m_vrs)
```

Comment:
This is exactly this check, because `operator bool()` for `optional` calls `is_initialized`